src/TileSlider/TileSlider.tsx

```diff
@@ -205,19 +220,6 @@ const TileSlider = <T extends unknown>({
     [minimalTouchMovement, slide, onSwipeStart, onSwipeEnd],
   );
 
-  useEffect(() => {
-    if (state.afterReset) {
-      if (frameRef.current) frameRef.current.style.transition = transitionBasis;
-      if (onSlideEnd) onSlideEnd();
```

Comment:
You're not calling this method anywhere now.

Reply:
Its still used in Symphony

Reply:
So you'll fix it? 😛